lib/rules/no-this-property-during-ssr.js

```diff
@@ -0,0 +1,377 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+'use strict';
+const { noPropertyAccessDuringSSR } = require('../rule-helpers');
+const { docUrl } = require('../util/doc-url');
+
+const disallowedProperties = [
```

Comment:
Only a subset of those APIs is implemented by `LightningElement` today. Could we be more selective?